bakery/src/scripts/disassemble_single.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+exec 2> >(tee disassembled-book/stderr >&2)
```

Comment:
disassembled-book should probably not be disassembled-book (should probably be some param)